meta/library/draft-1.json

```diff
@@ -89,28 +106,65 @@
             "protein expression"
           ]
         },
-        "link to paper": {
+        "Ontology type": {
+          "type": "string",
+          "description": "Type of onthology (TODO: Have an official list of ontologies)",
+          "enum":[
+            "pathway",
+            "cellular component",
+            "molecular component",
+            "biological process"
+          ]
+        },
+        "Organism": {
           "type": "string",
-          "format": "uri",
-          "description": "The link to the paper where this was published"
+          "pattern": "[A-Z][a-z]+ [a-z]+",
```

Comment:
This pattern looks fishy what are you looking to capture? This wouldn't, for instance, capture `Worm`

Reply:
It should capture organisms in scientific name format.